webview/src/plots/components/webviewSlice.ts

```diff
@@ -36,11 +38,16 @@ export const webviewSlice = createSlice({
     initialize: (state: { hasData: boolean }) => {
       state.hasData = true
     },
-    setMaxPlotSize: (
-      state: { maxPlotSize: number },
+    setSnapPoints: (
+      state: { snapPoints: SnapPoints },
       action: PayloadAction<number>
     ) => {
-      state.maxPlotSize = action.payload
+      state.snapPoints = [
```

Comment:
[Q] Can we make this a function to capture what is going on. E.g 
```
buildSnapPoints = (maxWidth: number): SnapPoints =>
```

(Took me a while to work it out)

Reply:
I should have read the description properly... but it won't be right there embedded in the code